### PR TITLE
Linux translate: Print status once

### PR DESCRIPTION
**Pull request by EricEdens**
At the end of Linux translation, statuses are double printed, once by Python and once by Bash. If Python reports a failure, but fails to flush its logs, then the incorrect status can be read by Daisy.

This was observed recently in the "Incorrect OS specified" test: 

```
 startup-script-url: TranslateStatus: <serial-output key:'detected_distro' value:'debian'>
 startup-script-url: TranslateStatus: <serial-output key:'detected_major_version' value:'9'>
 startup-script-url: TranslateStatus: <serial-output key:'detected_minor_version' value:'12'>
 startup-script-url: TranslateDebug: Traceback (most recent call last):
 startup-script-url:   File "/usr/local/lib/python3.7/dist-packages/utils/common.py", line 246, in RunTranslate
 startup-script-url:     translate_func()
 startup-script-url:   File "/usr/local/lib/python3.7/dist-packages/translate.py", line 320, in translate
 startup-script-url:     release = _get_release(g)
 startup-script-url:   File "/usr/local/lib/python3.7/dist-packages/translate.py", line 162, in _get_release
 startup-script-url:     product, major, minor, supported))
 startup-script-url: ValueError: Import of debian-9.12 is not supported. The following versions are supported: [opensuse-15.1|2, sles-15.1|2, sles-12.4|5]
 startup-script-url: 
 startup-script-url: TranslateFailed: error: Import of debian-9.12 is not supported. The following versions are supported: [opensuse-15.1|2, sles-15.1|2, sles-12.4|5]
 startup-script-url: + NotifyExit
 startup-script-url: + [ 0 -eq 0 ]
 startup-script-url: + logger -p daemon.info TranslateSuccess: exits with return code 0
Nov 12 13:30:33 inst-translator-import-image-gzzm6 root: TranslateSuccess: exits with return code 0
 startup-script-url exit status 0
 Finished running startup scripts.
```

This PR disables Bash's final-status printing for Linux translations (except if there's a failure prior to running the Python interpreter).

## Testing

* Ran the tests `cli_tools_tests/e2e/gce_image_import_export/test_suites/import/import_tests.go`

### CODE REVIEW

File: daisy_workflows/linux_common/bootstrap.sh [hunk 1]
```diff
@@ -131,4 +131,11 @@ chmod +x "$path_script"
 CheckPython3Installation
 
 logger -p daemon.info "Status: Running $path_script"
+# Remove the trap when script_prints_status equals "yes".
+# Useful when the script handles its own communication with
+# daisy (typically with prefixes such as "TranslateSuccess"
+# and "TranslateFailure").
+GetMetadataAttribute "script_prints_status"
```
Comment (dntczdx) [inline daisy_workflows/linux_common/bootstrap.sh:138]: it looks like "bootstrap.sh" is the only place to use this mechanism, however the attr name "script_prints_status" is very general.

Comment (EricEdens) [reply daisy_workflows/linux_common/bootstrap.sh:138]: The term "script" references the script that `boostrap.sh` will be executing. This is what it looks like in a workflow file: 

image

Comment (EricEdens) [reply daisy_workflows/linux_common/bootstrap.sh:138]: In other words, it's an attribute of the parameter `script`, rather than `bootstrap.sh`.

Comment (dntczdx) [reply daisy_workflows/linux_common/bootstrap.sh:138]: Cool

**Review by dntczdx — APPROVED** [review]

**Review by zoran15 — APPROVED** [review]

**Pull request merged**